libraries/provision/provision_cluster.py

```diff
@@ -45,6 +45,10 @@ def provision_cluster(couchbase_server_config, sync_gateway_config):
     status = ansible_runner.run_ansible_playbook("flush-firewall.yml")
     assert status == 0, "Failed to flush firewall"
 
+    # Set hostnames to match ansible hostnames
+    status = ansible_runner.run_ansible_playbook("set-hostnames.yml")
```

Comment:
To the best of my knowledge: no, the hostname is changed using the hostnamectl command.

The external DNS name does appear to changed (based on testing), although I have not restarted a VM after using hostnamectl, it's possible the DNS entry of the host would then change.

We should restart a VM that has had it's hostname changed, I'' sync up with @tleyden to test this.